scripts/populate_conferences.py

```diff
@@ -544,6 +684,14 @@ def populate_conferences():
                 admin_objs.append(user)
             except ModularOdmException:
                 raise RuntimeError('Username {0!r} is not registered.'.format(email))
+
+        # Convert string into datetime object
+        date_format = '%b %d %Y'
+        if attrs['end_date']:
+            attrs['end_date'] = datetime.strptime(attrs['end_date'], date_format)
```

Comment:
Nice. I haven't actually used `strptime` before (I usually just do plain old string parsing on dates). Good to know about! :)